docs/auth0_logs_list.md

```diff
@@ -31,7 +31,7 @@ auth0 logs list [flags]
 ```
   -f, --filter string   Filter in Lucene query syntax. See https://auth0.com/docs/logs/log-search-query-syntax for more details.
       --json            Output in json format.
-  -n, --number int      Number of log entries to show. (default 100)
+  -n, --number int      Number of log entries to show. Minimum 1, maximum 1000. (default 100)
```

Comment:
Why is the maximum only 1000?

That greatly reduces the utility of this in my mind.